packages/flutter/test/material/theme_test.dart

```diff
@@ -47,6 +47,74 @@ void main() {
     expect(Theme.of(tester.element(find.text('menuItem'))).brightness, equals(Brightness.dark));
   });
 
+  group('Theme.brightnessOf', () {
+    testWidgets('return correct brightness when just media query is given', (
+      WidgetTester tester,
+    ) async {
+      await tester.pumpWidget(
+        MediaQuery(
+          data: const MediaQueryData(platformBrightness: Brightness.dark),
+          child: Container(),
```

Comment:
I would use a `SizedBox`, instead of a `Container` to make the tests a little lighter.